index.js

```diff
@@ -1,11 +1,29 @@
-function scuberGreetingForFeet(){
-  // Write your code here!
+function scuberGreetingForFeet(ride){
+  // Write your code here!  
+  if (ride <= 400){
+    return 'This one is on me!';
+  } else if (ride >= 400 && ride <= 2000){
+    return 'That will be twenty bucks.'
+  } else if (ride > 2000 && ride < 2500){
+    return 'I will gladly take your thirty bucks.'
+  } else if (ride > 2500) {
+    return 'No can do.'
+  }
 }
 
-function ternaryCheckCity(){
+function ternaryCheckCity(trip){
   // Write your code here!
+  return trip === 'NYC'?'Ok, sounds good.' : 'No go.';
```

Comment:
Nit pick on spacing, but it matters for readability. 
```suggestion
  return trip === 'NYC' ? 'Ok, sounds good.' : 'No go.';
```